src/main.rs

```diff
@@ -90,6 +101,33 @@ fn app(authors: &'static str, version: &'static str) -> clap::App<'static, 'stat
                 .help("Runs the command in test mode")
                 .hidden(true),
         )
+        .subcommand(
+            /* 
+            Initial subcommand format
+            fpm start-project <project-name> [project-path]
+                               -n or --name   -p or --path
+            Necessary <project-name> with Optional [project-path]
+            */
+            clap::SubCommand::with_name("start-project")
+                .about("Creates a template ftd project at the target location with the given project name")
+                .arg(
+                    clap::Arg::with_name("project-name")
+                        .short("n")
+                        .long("name")
+                        .required(true)
+                        .takes_value(true)
+                        .help("Project name")
+                )
+                .arg(
+                    clap::Arg::with_name("project-path")
+                        .short("p")
+                        .long("path")
+                        .takes_value(true)
+                        .default_value(".")
```

Comment:
The default is wrong. If the package name is `foo.com/bar`, the default folder name will be `foo.com/bar`.

Reply:
Okay package names can be urls, got it. Am I allowed to handle this default case manually instead of using the default_value() which is available in clap ?

Reply:
I am not sure if you can implement this requirement using `default_value()` alone. Do it in your handler function.